functions/src/index.ts

```diff
@@ -121,20 +110,48 @@ exports.deleteStaleGames = functions.https.onRequest((request, response) => {
         Firebase.firestore().collection("games")
```

Comment:
there should be a return here. if not it wont return a 0 or -1

Reply:
ahh word good call... why do we need to return anything btw? isnt the main purpose to send back a response?